docs/resources/Guild.md

```diff
@@ -521,6 +521,106 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
+| ---------------- | ------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| id               | snowflake                                                                                               | the id of the prompt                                   |
+| options          | array of [prompt option](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-option-structure) objects | the options available to the prompt                    |
+| title            | string                                                                                                  | the title of the prompt                                |
+| single_select    | boolean                                                                                                 | can only one option of the prompt be selected at time? |
+| required         | boolean                                                                                                 | is this prompt required in onboarding flow?            |
+| in_onboarding    | boolean                                                                                                 | is this prompt in the onboarding flow?                 |
+| type             | [prompt type](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-type)                                | the type of the prompt                                 |
+
+
+###### Prompt Option Structure
+
+| Field       | Type                | Description                                                                               |
+| ----------- | ------------------- | ----------------------------------------------------------------------------------------- |
+| id          | snowflake           | the id of the option                                                                      |
+| channel_ids | array of snowflakes | the channels opted into when this option is selected                                      |
+| role_ids    | array of snowflakes | the roles assigned when this option is selected                                           |
+| emoji_id    | ?snowflake          | the [emoji id](#DOCS_REFERENCE/image-formatting), if the emoji is custom                  |
+| emoji_name  | ?string             | the emoji name if custom, the unicode character if standard, or `null` if no emoji is set |
+| title       | string              | the title of the option                                                                   |
+| description | ?string             | the description of the option                                                             |
+
+
+###### Prompt Type
+
+| Value | Name            |
+| ----- | --------------- |
+| 0     | Multiple Choice |
+| 1     | Dropdown        |
```

Comment:
I don't see any option for dropdowns in the UI, is this an actual thing that a bot could encounter, or leftover prototype code?

Reply:
there will be in the near future